docker-compose.yml

```diff
@@ -16,6 +16,11 @@ services:
     ports:
       - "4566:4566"
       - "8081:8081"
+    healthcheck:
+      test: ["CMD", "curl", "localhost:4566"]
```

Comment:
if we're using curl we'd want it fail, like `curl --fail`, I think. also, might be worth adding a TODO to write a custom healthcheck

Reply:
yeah I think adding `--fail` makes sense. I didn't know that flag existed. I'll add a TODO for the custom health check.

Reply:
resolved [here](https://github.com/GSA/datagov-harvesting-logic/commit/3e5b54136e0aa81a9598fd0bfe906cc0a540b776)